README.md

```diff
@@ -18,10 +18,32 @@
 git clone https://github.com/konveyor/imagestream-migrate && cd imagestream-migrate
 ```
 
-### 2. Automation prerequisites
+### 2. Prerequisites
 
-#### Virtualenv 
- * Installing Virtualenv
+#### Expose registries
+A route is created for image registries on OpenShift 3 by default. No further action should be required.
+If for some reason the registry is no exposed instructions are provided in the [OpenShift Documentation](https://docs.openshift.com/container-platform/3.11/install_config/registry/securing_and_exposing_registry.html#exposing-the-registry)
+
+On OpenShift 4 registries are not exposed by default. To create a route run:  
+`oc patch configs.imageregistry.operator.openshift.io/cluster --patch '{"spec":{"defaultRoute":true}}' --type=merge`
+
+#### Install required software
+Options using dnf, pip, and pip/virtualenv are provided below.
+
+##### dnf
+`dnf install ansible python3-openshift`
+
+##### pip
+
+```
+pip3 install ansible==2.9.7 --user      # ansible 2.9
+pip3 install kubernetes==11.0.0 --user  # kubernetes module for ansible
+pip3 install openshift==0.11.2 --user   # openshift module for ansible
+pip3 install PyYAML==5.1.1 --user       # pyyaml module for python
+```
+
+##### Virtualenv
+ * Install Virtualenv
```

Comment:
does anyone need to use this? this was just copied over, we can remove this section if unused

Reply:
Sure, I can definitely understand the need for pip since some are using Mac's, etc. but I don't think virtualenv is needed? I'll remove it.

Reply:
Removed.